csharp/ql/src/Likely Bugs/ThreadUnSafeICryptoTransformGood.cs

```diff
@@ -0,0 +1,38 @@
+internal class TokenCacheThreadUnsafeICryptoTransformDemo
```

Comment:
This file should be named `ThreadUnSafeICryptoTransformBad.cs`.

Reply:
Fixed it.